platform/src/main/java/org/stellar/anchor/platform/event/EventProcessorManager.java

```diff
@@ -0,0 +1,163 @@
+package org.stellar.anchor.platform.event;
+
+import static org.stellar.anchor.event.EventService.*;
+import static org.stellar.anchor.util.Log.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import lombok.SneakyThrows;
+import org.stellar.anchor.api.event.AnchorEvent;
+import org.stellar.anchor.api.exception.AnchorException;
+import org.stellar.anchor.event.EventService;
+import org.stellar.anchor.event.EventService.EventQueue;
+import org.stellar.anchor.platform.config.CallbackApiConfig;
+import org.stellar.anchor.platform.config.ClientsConfig;
+import org.stellar.anchor.platform.config.EventProcessorConfig;
+import org.stellar.anchor.platform.utils.DaemonExecutors;
+import org.stellar.anchor.util.Log;
+
+public class EventProcessorManager {
+  public static final String CLIENT_STATUS_CALLBACK_EVENT_PROCESSOR_NAME_PREFIX =
+      "client-status-callback-";
+  public static final String CALLBACK_API_EVENT_PROCESSOR_NAME = "callback-api";
```

Comment:
Typo: `callback-api-`

Reply:
Don't know how you catch this, but awesome catch.